course/lesson-5-your-first-function/lesson.tres

```diff
@@ -39,9 +39,9 @@ text = "In computer programming, we talk about [i]identifiers[/i] rather than \"
 
 It is because a function name is a label the computer uses to precisely [i]identify[/i] and refer to a function or other code elements.
 
-Identifiers are unique: you cannot reuse the same name in a given [i]space[/i] in your code.
+Identifiers are unique: you cannot reuse the same name in a given [i]script[/i] in your code.
 
-If you try to name two functions the same, the computer will raise an error."
+If you try to name two functions the same, in the same script, the computer will raise an error."
```

Comment:
Same, shouldn't talk about script as students aren't supposed to know what this is